templates/list_pages/node--oe-list-page--full.html.twig

```diff
@@ -25,10 +25,14 @@
     <div class="col-12 col-lg-9 col-xxl-8">
       <div class="row">
         <div class="col-md-6 col-lg-8 align-self-center">
-          <h4 class="mb-0">
-            <span class="text-capitalize">{{ label }}</span>
-            ({{ content['extra_field_oe_list_page_total_count:node:oe_list_page'] }})
-          </h4>
+          {% set _title %}
+            {{ label|render|striptags }}({{ content['extra_field_oe_list_page_total_count:node:oe_list_page'] }})
+          {% endset %}
+          {% include '@oe-bcl/bcl-heading/heading.html.twig' with {
+            title: _title,
+            title_tag: 'h2',
+            attributes: create_attribute().addClass(['mb-0', 'text-capitalize']),
+          } only %}
```

Comment:
Btw this is one of the candidates to handle in a follow-up issue.
It just feels wrong to me to split it out now that we already touched it.